src/lib/pickles_types/pcs_batch.mli

```diff
@@ -46,15 +46,15 @@ val combine_evaluations' :
   -> ('f, 'm) Vector.t
   -> 'f
 
-open Plonk_types.Poly_comm
-
 val combine_split_commitments :
      (_, 'n, 'm) t
   -> scale_and_add:(acc:'g_acc -> xi:'f -> 'g -> 'g_acc)
   -> init:('g -> 'g_acc option)
   -> xi:'f
-  -> ('g Without_degree_bound.t, 'n) Vector.t
-  -> ('g With_degree_bound.t, 'm) Vector.t
+  -> reduce_without_degree_bound:('without_degree_bound -> 'g list)
```

Comment:
Oh, some new parameters! And in an interface! That would be a good time to start documenting the function, and describing what the parameters are supposed to do :).

Reply:
In theory yes, but now is not that 'good time'.